ApollosProject/apollos-prototype
# Fixes wrong birthdate showing up in the datepicker.

![2019-08-01 16 05 00](https://user-images.githubusercontent.com/1637694/62323825-23b95580-b476-11e9-9cb1-3688e52fd1ca.gif)
## DESCRIPTION

### What does this PR do, or why is it needed?

When you view the datepicker, it will show your correct birthdate. Opening the datepicker will default to an incorrect birthdate, however. 

The fix for this is two-pronged. - send over the birthdate from the API with the timezone properly baked in, and don't switch dates defined in eastern into UTC when showing a datepicker date (potentially causing rollover issues) 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Open the onboarding experience
2. Visit the "About You" tab.
3. Check your birthdate in the button. it should be correct.
4. Tap the button and check the default birth date listed, it should match the date in the button 

### What automated tests did you write?

updated snaps

## TODO:

- [x] Closes #942 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
